proto/identity/associations/association.proto

```diff
@@ -91,6 +91,7 @@ message AssociationState {
   repeated MemberMap members = 2;
   string recovery_address = 3;
   repeated bytes seen_signatures = 4;
+  uint64 created_at = 5;
```

Comment:
We'd want to have the created_at field on each member in the `MemberMap`

Reply:
Got it